packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardImage.styles.ts

```diff
@@ -0,0 +1,63 @@
+import { getGlobalClassNames } from '../../Styling';
+import { IDocumentCardImageStyleProps, IDocumentCardImageStyles } from './DocumentCardImage.types';
+
+export const GlobalClassNames = {
+  root: 'ms-DocumentCardImage',
+  icon: 'ms-DocumentCardImage-icon',
+  centeredIcon: 'ms-DocumentCardImage-centeredIcon',
+  centeredIconContainer: 'ms-DocumentCardImage-centeredIconContainer',
+  cornerIcon: 'ms-DocumentCardImage-cornerIcon'
+};
```

Comment:
Preferably I'd not add more global classnames and let consumers of the component use `styles` to apply custom styling.

But perhaps someone from the Fabric Team can comment :)

Reply:
Yeah, was wondering about that. Definitely prefer not adding if not necessary

Reply:
@mikewheaton, @dzearing - Should we keep the global classnames, or ensure we dont introduce new ones?

Reply:
I think for now we're still adding global class names for each region of a component. I'd be fine with leaving them out though, as they could always be added in a later PR if someone needs them.